packages/atomic/src/components/common/generated-answer/show-button.tsx

```diff
@@ -0,0 +1,31 @@
+import {FunctionalComponent, h} from '@stencil/core';
+import {i18n} from 'i18next';
+import ArrowBottomIcon from '../../../images/arrow-bottom-rounded.svg';
+import ArrowTopIcon from '../../../images/arrow-top-rounded.svg';
+import {Button} from '../button';
+
+interface ShowButtonProps {
+  i18n: i18n;
+  isCollapsed: boolean;
+  onClick(): void;
```

Comment:
I'd suggest to write it this way. Both patterns are used in the project, but I see more occurrences of this one.

```suggestion
  onClick: () => void;
```